### DISCUSSION TITLE
有没有可能做个自动化脚本来维护的卡牌图鉴

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#973] [Ideas]
1. 有稳定的 API 能扒数据和卡图
2. 再压榨 @Mark9804 老大哥一把，画个网页，我们一般行动很慢，其他项目应该会有现成的网页可以扒过来

先观望一下

**Comment by mark9804** [comment]
我等了 enka API 半年是我能说的吗

**Reply by Arondight** [reply]
这玩意儿我还没看，逐渐现充了，没有大把时间了

**Reply by Arondight** [reply]
另外至少米游社有一个基本的信息展示， enka 算是个上位替代，拿到数据后角色的展示得像模像样用 projects 管理一下才能协调好开发任务，卡牌这个东西，一般 api 和功能都很简单，做起来应该有戏

**Comment by Arondight** [comment]
ambr 没有这功能，搁置了吧

**Comment by mark9804** [comment]
希望 ambr 比我们勤快
![c80989f058f3722efb46adc270d178ad](https://user-images.githubusercontent.com/9006264/208284862-edc7fa02-fc4e-4e8e-a249-b5ee9bc9f6e4.gif)

**Comment by mark9804** [comment]
https://github.com/Nwflower/genshin-atlas
